foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-MySQLPolicySetDefinition.json

```diff
@@ -17,47 +17,538 @@
+                                                                    "enabled": "[parameters('metricsEnabled')]",
+                                                                    "retentionPolicy": {
+                                                                        "days": 0,
+                                                                        "enabled": false
+                                                                    },
+                                                                    "timeGrain": null
+                                                                }
+                                                            ],
+                                                            "logs": [
+                                                                {
+                                                                    "categoryGroup": "allLogs",
+                                                                    "enabled": true
+                                                                }
+                                                            ]
+                                                        }
+                                                    }
+                                                ],
+                                                "outputs": {}
+                                            },
+                                            "parameters": {
+                                                "logAnalytics": {
+                                                    "value": "[parameters('logAnalytics')]"
+                                                },
+                                                "location": {
+                                                    "value": "[field('location')]"
+                                                },
+                                                "resourceName": {
+                                                    "value": "[field('name')]"
+                                                },
+                                                "profileName": {
+                                                    "value": "[parameters('profileName')]"
+                                                },
+                                                "metricsEnabled": {
+                                                    "value": "[parameters('metricsEnabled')]"
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "name": "Dine-Diagnostics-MySql"
+                },
+                {
+                    "properties": {
+                        "displayName": "Deny MySQL servers without customer-managed keys to encrypt data at rest",
```

Comment:
This will not work as we are looking at a child resource (keys) that can only exist if used with Azure Key Vault. The correct enforcement value would be DINE to ensure /keys is created for the Servers resourceType. However, assigning this at scale is out of scope as the URI of the (existing) key must be specified.